Assets/Plugins/Source/Core/EOSManager_DynamicLoading.cs

```diff
@@ -67,34 +67,30 @@ public partial class EOSSingleton
 
             public const string EOSBinaryName = Epic.OnlineServices.Config.LibraryName;
 
+
 #if USE_EOS_GFX_PLUGIN_NATIVE_RENDER
-//#if UNITY_WSA
```

Comment:
Not sure if this is disabled temporarily or just not used anymore.

Reply:
Pretty sure it's not used anymore - looking at the rest of the file (for instance `static extern void UnloadEOS()` on line `85`).

Reply:
In fact - I just looked through the history - these lines have been commented out for over a year now - so I'm confident they can be removed.